docs/Agent/installation/rpm_based_installation.md

```diff
@@ -22,9 +22,9 @@ In the following, we describe how to install Pbench Agent using an RPM.
 	```
 
     :::{note}
-	- This documentation source describes Pbench Agent 0.72, and here's where you find the RPMs [ndokos/pbench-0.72](https://copr.fedorainfracloud.org/coprs/ndokos/pbench-0.72).
-	- There are some RPMs that are shared between versions (e.g. pbench-sysstat). We maintain those in the [ndokos/pbench](https://copr.fedorainfracloud.org/coprs/ndokos/pbench)
-    - On RHEL based system enable subscription manager & also enable EPEL repo.
+	- This documentation source describes Pbench Agent 0.72, and [here's](https://copr.fedorainfracloud.org/coprs/ndokos/pbench-0.72) where you find the RPMs.
```

Comment:
The `[here's]` link bothers me; I realize that it's because when you're just *reading* the rendered text, the link text means absolutely nothing unless you hover or click to figure out where it leads. (And if someone were to print it out, they couldn't even have that much of a clue.)

I don't think we want the link text to be the full URL; however saying that it's the `ndokos` COPR `pbench-0.72` repo would be more useful ... the reader then has some hope of finding it (or at least understanding the context) without the link!

Maybe `[ndokos/pbench-0.72]` like what you have before to the non-release RPMs...

Reply:
This is an interesting point.  The sample commands at lines 20-21 are very concrete:  they make it clear that the RPMs come from COPR from the `ndokos/pbench-0.72` and `ndokos/pbench` repos.

So, presumably what we need to do in this note is to make it clear to the user/admin that they should be selecting the appropriate repo for their version (e.g., they should replace the `0.72` with the desired version).

It's probably not appropriate for our documentation to try to explain what the DNF `copr` plug-in does -- that is, we probably shouldn't have hyperlinks to `copr.fedorainfracloud.org` -- the user will either know that already or they can use external sources to learn about DNF and COPR (rather than duplicating that information here and possibly letting it get stale).

So, I think I would suggest simply removing the text which Dave cited here.  (E.g., presumably, the user won't be using the hyperlink reference to obtain the RPMs -- that will be done automatically after enabling the COPR repos.)

If we really want some text like this here, it should probably just mention that the user should substitute the desired version of the Pbench Agent in place of "0.72".

Reply:
The central "interesting point", I think, is a question: how do we tread the knife-edge between "generic installation documentation" and "release documentation". Although we're mentioning "0.72" specifically here, arguably that's more appropriate for the 0.72 release document, because we shouldn't necessarily need to update this every release.

So, yeah: maybe just a general statement about how we structure the COPR repo names, (`pbench-<release>`, maybe even "for example, `pbench-0.72`") under the `ndokos` account. Actual hyperlinks are probably not useful, unless perhaps just to the `ndokos` account where they could browse.